app/models/cohort.rb

```diff
@@ -3,7 +3,6 @@
 class Cohort < ApplicationRecord
   has_paper_trail
 
-  NPQ_PLUS_1_YEAR = 2020
```

Comment:
I wonder if instead of removing this we should rename it, as not sure we should remove this logic 🤔 let me know you're thoughts

Reply:
We're only using this for `npq_plus_one_or_earlier?` and used in `app/forms/schools/add_participants/base_wizard.rb`

Reply:
Since we are using it in the base wizard I wonder if we keep this logic which says if the cohort is equal to or less than 2020 we default to the current cohort, since it's used on the ECF side. Just as an extra guard maybe incase there is some bad ecf data somewhere?

Reply:
Reverted the change. I renamed the method `npq_plus_one_or_earlier?` to `start_year_2020_or_earlier?`
Let me know what you think.